src/duckstation-qt/consolesettingswidget.cpp

```diff
@@ -194,7 +194,6 @@ void ConsoleSettingsWidget::onEnableCPUClockSpeedControlChecked(int state)
 
     QMessageBox* const mb = QtUtils::NewMessageBox(QMessageBox::Warning, tr("CPU Overclocking Warning"), message,
                                                    QMessageBox::NoButton, QMessageBox::NoButton, Qt::WindowModal, this);
-    mb->setAttribute(Qt::WA_DeleteOnClose, true);
```

Comment:
This should not be removed. Might be easier to just merge and re-add it.

Reply:
I just moved it to `QtUtils::NewMessageBox()`